src/button-fixed-footer-layout.tsx

```diff
@@ -12,8 +10,6 @@ import type {RendersNullableElement} from './utils/types';
 type Props = {
     isFooterVisible?: boolean;
     button?: NullableButtonElement;
-    /** @deprecated */
-    desktopButtonAlign?: 'left' | 'center';
```

Comment:
I'm not sure what's the expected behaviour in desktop if we don't allow this. As of now, it will be equivalent to mobile (`full-width`). I've checked in webapp and there are only two cases in which it's being used with `center`, and I don't know why

Reply:
I wouldn't mind. Those are not production cases, so not a big deal if the buttons change alignment in desktop